cmd/kubeadm/app/images/images_windows.go

```diff
@@ -0,0 +1,32 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+)
+
+// GetPauseImage returns the image for the "pause" container
+func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
+	const PauseImageRepository = "mcr.microsoft.com/k8s/core"
+	const PauseImage = "pause"
+	const PauseImageVersion = "1.2.0"
```

Comment:
Latest windows pause image for 1903/1909 support is `mcr.microsoft.com/oss/kubernetes/pause:1.3.0`
 See https://github.com/kubernetes/kubernetes/issues/87339#issuecomment-582136209

Reply:
this can be formed like so:
```
func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
	return GetGenericImage(
		"mcr.microsoft.com/k8s/core"
		"pause",
		"1.3.0"
	)
}
```

we discussed that hardcoding the image repository will be needed on windows for now.

Reply:
Please also note the path change `mcr.microsoft.com/oss/kubernetes` not `mcr.microsoft.com/k8s/core`

Reply:
Thanks for the catch!